erpnext/patches/v13_0/add_default_project_into_project_details_child_table.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doc('projects', 'doctype', 'task_project')
+	frappe.reload_doc('projects', 'doctype', 'task')
+
+	tasks = frappe.db.get_all("Task", fields=["*"])
+	for task in tasks:
+		if task.project:
+			doc = frappe.get_doc("Task", task.name)
+			doc.append("projects", {
+				"project": task.project,
+				"status": task.status
+			})
+			doc.save()
```

Comment:
- This will be a long and heavy patch if there are too many tasks.

Reply:
so what will be solution